Source/DafnyTestGeneration/ProgramModifier.cs

```diff
@@ -268,7 +268,7 @@ private class AnnotationVisitor : StandardVisitor {
         node.Blocks[0].cmds.Insert(0, GetAssumeCmd(types));
 
         // record parameter values:
-        var values = new List<string> { "\"Impl\"", $"\"{implName}\"" };
+        var values = new List<string> { "\"Impl\"", $"\"{node.VerboseName.Split(" ")[0]}\"" };
```

Comment:
I don't understand this part

Reply:
Also from the other PR, but an earlier version. You can see a discussion of it [here](https://github.com/atomb/dafny/pull/1#discussion_r900251083).

Reply:
The idea is to get the fully qualified name of a Dafny method from Boogie. Doing this allows getting rid of [this method](https://github.com/dafny-lang/dafny/blob/479bdda4f676bf12d743c15e03db1c43dc05d9f6/Source/DafnyTestGeneration/Utils.cs#L34-L47) so I think it should make things much more stable.